packager/app/packager_main.cc

```diff
@@ -325,9 +325,17 @@ base::Optional<PackagingParams> GetPackagingParams() {
   ChunkingParams& chunking_params = packaging_params.chunking_params;
   chunking_params.segment_duration_in_seconds = FLAGS_segment_duration;
   chunking_params.subsegment_duration_in_seconds = FLAGS_fragment_duration;
+  chunking_params.is_low_latency_dash = FLAGS_is_low_latency_dash;
   chunking_params.segment_sap_aligned = FLAGS_segment_sap_aligned;
   chunking_params.subsegment_sap_aligned = FLAGS_fragment_sap_aligned;
 
+  if (chunking_params.is_low_latency_dash && chunking_params.subsegment_duration_in_seconds > 0) {
+    LOG(ERROR) << "Fragment duration --fragment_duration, "
+                  "cannot be specified if LL-DASH --is_low_latency_dash, "
```

Comment:
nit: I'm not convinced that these commas make sense.

Also, why can't fragment_duration be used with is_low_latency_dash?  If the explanation is too long for the error log (which I would assume to be the case), then perhaps you could put an explanation in a comment just above the error log.  Then anyone researching the error would find the full explanation in the source.

Reply:
I added a clarifying comment above the error log. 

Also, those commas hurt my eyes, brain, and soul. I think I was going for a ", clarification," kinda thing, but forgot the first comma? Yikes.